enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -1,4 +1,51 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  gitRawCommitsOpts: { grep: "(sdk)" },
+  gitRawCommitsOpts: {
+    grep: "(sdk)",
+  },
+  options: {
+    preset: "angular",
+    pkg: {
+      path: "enterprise/frontend/src/embedding-sdk/package.template.json",
+    },
+    tagPrefix: "embedding-sdk-",
+    transform: function (commit, cb) {
+      if (typeof commit.gitTags === "string") {
+        const tagPrefix = "embedding-sdk-";
+        const rtag = new RegExp(`tag:\\s*[=]?${tagPrefix}(.+?)[,)]`, "gi");
+        const match = rtag.exec(commit.gitTags);
+        rtag.lastIndex = 0;
+        if (match) {
+          commit.version = match[1];
+        }
+      }
+      if (commit.committerDate) {
+        const dateFormatter = Intl.DateTimeFormat("sv-SE", {
```

Comment:
(looks like the Swedish (Sweden) datetime format is used here, but if it works then that's ok)

Reply:
Ah, I forgot to comment on this, this is the only other place that has changed since the original PR refers to this `dateFormatter` defined outside the `transform` function.

I use the exact same date formatter.

https://github.com/conventional-changelog/conventional-changelog/blob/65550d9a30cfab4b00b3e982e42bd68fedafcefc/packages/conventional-changelog-core/lib/merge-config.js#L13-L15